Kickstarter-iOS/Views/PaymentMethodsFooterView.swift

```diff
@@ -10,24 +10,25 @@ public final class PaymentMethodsFooterView: UITableViewHeaderFooterView {
 
   public weak var delegate: PaymentMethodsFooterViewDelegate?
 
-  @IBOutlet private weak var plusLabel: UILabel!
-  @IBOutlet private weak var titleLabel: UILabel!
+  @IBOutlet private weak var addCardButton: UIButton!
   @IBOutlet private weak var separatorView: UIView!
 
   override public func bindStyles() {
     super.bindViewModel()
 
-    _ = self
-      |> \.backgroundColor .~ .ksr_grey_200
-
-    _ = self.plusLabel
-      |> \.textColor .~ .ksr_green_700
-
-    _ = self.titleLabel
-      |> \.text %~ { _ in
+    _ = self.addCardButton
+      |> UIButton.lens.titleColor(for: .normal) .~ .ksr_green_700
```

Comment:
Should we prefer using `key paths` over `lenses`? Trying to refer to @Scollaco 's RFC

Reply:
So for things like `titleColor(for:` we can't use keyPaths because they're functions, but for `imageEdgeInsets` and `titleEdgeInsets` will do!